router/pkg/otel/attributes.go

```diff
@@ -78,3 +78,25 @@ var (
 	RouterServerAttribute    = WgComponentName.String("router-server")
 	EngineTransportAttribute = WgComponentName.String("engine-transport")
 )
+
+const (
+	ServerAddress = attribute.Key("server.address")
+	ServerPort    = attribute.Key("server.port")
+)
+
+func GetServerAttributes(host string) []attribute.KeyValue {
```

Comment:
According to semconv, there were a bunch of similar attributes that we could have used but all of them seem to have been deprecated in favour of `server.address` and `server.port`, thus we have to split the port and host component.

In case a port is not provided / bad host, we stick to returning only the host component.

cc: @StarpTech